moss-client/moss-client-common/pom.xml

```diff
@@ -70,6 +70,40 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.9.9</version>
         </dependency>
+        <dependency>
```

Comment:
You don't need all of these dependencies, you're not using all of them in the code.